### PR TITLE
fix min tls setting for webhook server (#1225)

### PR DESCRIPTION
**Pull request by kangsheng89**
Fix issue #1225

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -151,10 +156,16 @@ func main() {
...
+
+	optionsTlSOptsFuncs := []func(*tls.Config){
+		func(config *tls.Config) { minTlsDefault(config) },
+	}
+
 	mgrOptions := ctrl.Options{
 		Scheme:                 scheme,
 		MetricsBindAddress:     metricsAddr,
 		Port:                   webhookPort,
+		TLSOpts:                optionsTlSOptsFuncs,
```
Comment (pavolloffay) [inline main.go:168]: Does this configure the TLS version on the webhook server as well? 


```
mgrOptions := ctrl.Options{
		WebhookServer:
```

Comment (kangsheng89) [reply main.go:168]: yes, i tested it on my local, and i check from sslscan. Later i attached printscr here

**Comment by kangsheng89** [comment]
@pavolloffay can help to approve for running workflow?

**Comment by kangsheng89** [comment]
image
This is before change, where i portforward port from 9443 to localhost:10001

**Comment by kangsheng89** [comment]
image
This is after fix, with portforward from 9443 to localhost:10001

As u can see, now the webhook server that run on port 9433 TLS setting enforce with v1.2 and above

**Comment by kangsheng89** [comment]
let me check the comment, i think should remove it to make it pass the golangci-lint

**Pull request merged**